src/style.css

```
@@ -1,3 +1,84 @@
+
 body {
+margin: 0;
```

```suggestion
  margin: 0;
```
Please add two spaces for the CSS properties of the selector, fix it for all the similar cases